keyboards/kinesis/kint36/chconf.h

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2020 QMK
```

Comment:
Are the majority of these necessary? Seems like the original config must've been with everything enabled?
Building locally after deleting this file ends up producing a binary -- might want to check if it can just be removed.

Reply:
chconf.h now minified. The remaining settings need to be present, otherwise the generated binary doesn’t work on the Teensy 3.6.